js/capture.js

```diff
@@ -19,13 +27,61 @@ const capture = {
       });
   },
 
+  // In the parsed whitelist JSON object (from disconnect-entitylist.json):
+  //   - 'properties' keys are first parties
+  //   - 'resources' keys are third parties
+  reformatList(whiteList) {
```

Comment:
I don't always do it, but when I leave myself [a docblock showing an example of these lists' data formats](https://github.com/mozilla-services/shavar-prod-lists/blob/master/scripts/json_verify.py#L68-L102), I always thank myself later when I have to come back to understand how a function reformats or transforms this data.

In this case, it looks like the function turns this:
```
"Facebook" : {
  "properties": [
    "facebook.com",
    "facebook.de",
    ...
    "messenger.com"
  ],
  "resources": [
    "facebook.com",
    "facebook.de",
    ...
    "akamaihd.net"
  ]
}
```
into this:
```
{
  {
    "facebook.com": 0,
    "facebook.de": 1,
    ...
    "messenger.com": 9
  },
  {
    0: [ "facebook.com", "facebook.de", ... "akamaihd.com"],
    1: [ "facebook.com", "facebook.de", ... "akamaihd.com"],
    ...
    9: [ "facebook.com", "facebook.de", ... "akamaihd.com"]
  }
}
```
?

Whatever the structure of the data you make it (the logic could probably be optimized for performance and accuracy per @jonathanKingston's comments), I suggest adding a doc-block in front of this function to make it easy to remember later.